### PR TITLE
Chore/create auth helpers

### PR DESCRIPTION
**Pull request by Wandarry**
Create authentification helpers with firebase

### CODE REVIEW

File: app/(tabs)/onboarding.tsx [hunk 1]
```diff
@@ -126,7 +128,9 @@ export default function Onboarding() {
               <SolidSmallWhite
                 message="Me connecter"
                 isDisabled={false}
-                onPress={() => console.log("Hello")}
+                onPress={() => {
```
Comment (BOCOVO) [inline app/(tabs)/onboarding.tsx]: @Wandarry 
I don't really understand what you are trying to do here.

File: app/(tabs)/onboarding.tsx [hunk 2]
```diff
@@ -8,7 +8,9 @@ import {
   AnimatedView,
   AnimatePresence,
 } from "@gluestack-style/animation-resolver";
+import ToastEventHandler from "@/components/ToastEventHandler";
 import { styled } from "@gluestack-style/react";
+import { ToastConfig } from "@/types/toast";
```
Comment (BOCOVO) [inline app/(tabs)/onboarding.tsx]: This seems not used

File: app/(tabs)/onboarding.tsx [hunk 3]
```diff
@@ -8,7 +8,9 @@ import {
   AnimatedView,
   AnimatePresence,
 } from "@gluestack-style/animation-resolver";
+import ToastEventHandler from "@/components/ToastEventHandler";
```
Comment (BOCOVO) [inline app/(tabs)/onboarding.tsx]: Is this necessary in this file ?

File: components/ToastEventHandler.tsx
Comment (BOCOVO) [inline components/ToastEventHandler.tsx:1]: This should not come here

**Review by BOCOVO — CHANGES_REQUESTED** [review]

File: app/(tabs)/onboarding.tsx (hunk 1 above)
Comment (Wandarry) [reply app/(tabs)/onboarding.tsx]: Sorry, I was trying to test toast here but ending creating another file while not removing this.

**Review by BOCOVO — APPROVED** [review]

**Pull request merged**